### PR TITLE
[DOXIATOOLS-80] Update to Doxia 2.0.0-M6

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

**Comment by kwin** [comment]
@michael-o Any idea why one XHTML5 test fails with a parsing exception in Doxia 2?

File: pom.xml [hunk 1]
```diff
@@ -173,17 +123,35 @@ under the License.
...
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.36</version>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>1.2.11</version>
+    </dependency>
```
Comment (michael-o) [inline pom.xml:135]: Why not SLF4J simple with runtime scope?

File: src/main/java/org/apache/maven/doxia/DefaultConverter.java [hunk 2]
```diff
@@ -370,23 +335,21 @@ private void parse(
         File outputDirectoryOrFile = relativeOutputDirectory != null
                 ? new File(output.getFile(), relativeOutputDirectory.getPath())
                 : output.getFile();
-        if (getLog().isDebugEnabled()) {
-            getLog().debug("Parsing file from '" + inputFile.getAbsolutePath() + "' with the encoding '"
-                    + inputEncoding + "' to '" + outputDirectoryOrFile.getAbsolutePath()
-                    + "' with the encoding '" + output.getEncoding() + "'");
-        }
+        LOGGER.debug(
+                "Parsing file from '{}' with the encoding '{}'" + " to '{}' with the encoding '{}'",
```
Comment (michael-o) [inline src/main/java/org/apache/maven/doxia/DefaultConverter.java:339]: Why not make it one string now?

File: src/main/java/org/apache/maven/doxia/cli/ConverterCli.java [hunk 3]
```diff
@@ -100,11 +98,9 @@ private static int doMain(String[] args) {
         }
 
         Converter converter = new DefaultConverter();
-        Log log = new SystemStreamLog();
         if (debug) {
-            log.setLogLevel(Log.LEVEL_DEBUG);
+            // TODO: programmatically adjust log level
         }
```
Comment (michael-o) [inline src/main/java/org/apache/maven/doxia/cli/ConverterCli.java:103]: Why then not drop the empty `if` clause?

**Review by michael-o — CHANGES_REQUESTED** [review]

**Comment by michael-o** [comment]
> @michael-o Any idea why one XHTML5 test fails with a parsing exception in Doxia 2?

I guess it misses the `<main />` element. Look at what Doxia 2 now produces, this must match.

**Comment by kwin** [comment]
The generated XHTML is valid (look at the generated file) still the parser stumbles.

File: pom.xml (hunk 1 above)
Comment (kwin) [reply pom.xml:135]: fixed

**Review by slachiewicz — APPROVED** [review]

**Review by michael-o — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #109

**Pull request merged**